liquidctl/driver/commander_pro.py

```diff
@@ -250,7 +266,7 @@ def initialize(self, direct_access=False, **kwargs):
             if self._hwmon:
                 _LOGGER.warning('forcing re-initialization despite %s kernel driver',
                                 self._hwmon.driver)
-            return self._initialize_directly()
+            return self._initialize_directly(fan_modes)
```

Comment:
What about when using a hwmon driver?

At least a warning might be appropriate if we're going to ignore `fan_mode` in that case.

Reply:
fixed in 522976b